src/app/globals.css

```
@@ -1,10 +1,18 @@
 @import "tailwindcss";
 
+
 :root {
   --background: #ffffff;
   --foreground: #111111;
 }
 
+@theme {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)